scripts/ci/debian/mono-llvm.install

```diff
@@ -0,0 +1,1574 @@
+usr/lib/libLLVMAMDGPUAsmParser.a
```

Comment:
Mono currently relies on our llvm being the one in the PATH and LDPATH. If we move where it ends up, we will need to make this change in mono as well.